org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -318,3 +318,44 @@ $button-border-radius: 2px;
     }
   }
 }
+
+%btn-primary {
+  background: $brand-primary !important;
+  border: none;
+  color: #fff !important;
+
+  &:hover {
+    background: darken($brand-primary, 10%) !important;
+
+    > input.crm-form-submit {
+      border: 1px solid darken($brand-primary, 10%) !important;
+    }
+  }
+
+  > input.crm-form-submit {
```

Comment:
So you are implicitly expecting that whatever is extending `%btn-primary` will contain a `.crm-form-submit`? Seems very specific. Could you give here an example of what element you are using this on?